save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -48,38 +48,50 @@ class FixPlugin(
 
         val fixPluginConfig = testConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
         val generalConfig = testConfig.pluginConfigs.filterIsInstance<GeneralConfig>().singleOrNull()
-        return files
-            .map { it.first() to it.last() }
-            .map { (expected, test) ->
-                val testCopy = createTestFile(test)
-                val execCmd = "${(generalConfig!!.execCmd)} ${fixPluginConfig.execFlags} $testCopy"
-                val executionResult = try {
-                    pb.exec(execCmd, null)
-                } catch (ex: ProcessExecutionException) {
-                    return@map TestResult(
-                        listOf(expected, test),
-                        Fail("${ex::class.simpleName}: ${ex.message}", ex::class.simpleName ?: "Unknown exception"),
-                        DebugInfo(null, ex.message, null)
-                    )
-                }
-                val stdout = executionResult.stdout.joinToString("\n")
-                val stderr = executionResult.stderr.joinToString("\n")
 
+        return files.chunked(fixPluginConfig.batchSize ?: 1).map { chunk ->
+            val pathMap = chunk.map { it.first() to it.last() }
+            val pathCopyMap = pathMap.map { (expected, test) -> expected to createTestFile(test) }
+            val testCopyNames = pathCopyMap.joinToString { (_, testCopy) -> testCopy.toString() }
```

Comment:
I'm concerned about separator for file names, both here and in warn-plugin. I think usually multiple CLI arguments are either separated by a whitespace or comma, not `", "`, like the default separator. I think we should make it configurable, probably in a separate PR.

Reply:
Yes, I think the separator should be customizable, too

Reply:
#158